### ISSUE TITLE
Two page layout for tablets

### ISSUE THREAD

**Issue by stopiccot** [FolioReader/FolioReaderKit#200]
It would be nice to have a two page layout options for landscape orientation on tablets:

![img_0008](https://cloud.githubusercontent.com/assets/24836/21604647/64e47f40-d1b5-11e6-8b3a-91cc4f5e330f.PNG)

**Comment by hebertialmeida** [comment]
This is pretty interesting, but I have no idea how to accomplish this, as we are already using the webView pagination.

Need to look deeper on that.

**Comment by HadrienGardeur** [comment]
Also, it's worth pointing out that there are two very different cases in EPUB:

- if the publication is a Fixed Layout EPUB, you'll actually display two different resources (probably using two different webviews) and will have to look into the way EPUB handles spreads (left/right page, orientation, double pages)
- for a reflowable EPUB, this is indeed tied to how you actually paginate the content in a single webview

**Comment by zh-wowtv** [comment]
Yes, it would be very nice to implement it like what iBook does.
